src/utils/minify.js

```diff
@@ -0,0 +1,75 @@
+// @flow
```

Comment:
This code was taken from [babel-plugin-styled-components](https://github.com/styled-components/babel-plugin-styled-components/blob/master/src/minify/index.js) and slightly reduced as ASTs aren't really a concern here.